activity-stream/simple-snippet.html

```diff
@@ -0,0 +1,134 @@
+{#
+Simple Snippet for Activity Stream with button option.
+
+Variables:
+
+  @blockable: checkbox - Check to allow user to block this snippet.
+  @rtl: checkbox, optional - If checked change text direction to support RTL languages
+  @icon: image, optional - Snippet icon. 64x64px. SVG or PNG preferred.
+  @button_label: text, optional - Text for a button next to main snippet text that links to button_url. Requires button_url.
+  @button_url: text, optional - A url, button_label links to this.
+  @text: text - Text of snippet.
+  @title: text, optional - Snippet title displayed above snippet text.
+
+#}
+<style>
+  /* This should probably be moved out of the template. */
+  #snippets-container {
+    border-top: solid 2px #E9E9E9;
+    background: rgba(255, 255, 255, 0.97);
+  }
+
+  #snippet {
+    margin: 28px 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    {% if rtl %}
+      flex-direction: row-reverse;
+    {% else %}
+      flex-direction: row;
+    {% endif %}
+  }
+
+  .snippet section {
+    width: 420px;
+    color: #383E49;
+    {% if rtl %}
+      direction: rtl;
+    {% else %}
+      direction: ltr;
+    {% endif %}
+    font-size: 12px;
+  }
+
+  .snippet h1 {
+    line-height: 20px;
+    font-weight: bold;
+    margin: 0;
+    font-size: inherit;
+  }
+
+  .snippet p {
+    margin: 0;
+    line-height: 16px;
+    font-size: inherit;
+  }
+
+  .snippet img {
+    height: 64px;
+    width: 64px;
+    {% if rtl %}
+      margin-left: 15px;
+    {% else %}
+      margin-right: 15px;
+    {% endif %}
+  }
+
+  .snippet section p a {
+    text-decoration: underline;
+  }
+
+  .snippet .button-link {
+    white-space: nowrap;
+    {% if rtl %}
+      margin-right: 15px;
+    {% else %}
+      margin-left: 15px;
+    {% endif %}
+    background: #9CA2A9;
+    color: #FFFFFF;
+    font-size: 14px;
+    padding: 10px 28px;
+    border-radius: 6px;
+    box-shadow: 0 2px 0 #74777C;
+  }
+
+  .snippet .button-link:hover {
+    color: #FFF;
+  }
+
+  {% if blockable %}
```

Comment:
What do you think about adding this css to the global snippets css (see https://github.com/mozmeao/snippets-service/pull/260) and bring this down to

```
  {% if blockable %}
    .snippet .block-snippet-button {
      {% if rtl %}
        left: 50px;
      {% else  %}
        right: 50px;
      {% endif %}

    }
  {% endif %}
```

This reduce the size of the payload and the complexity of the templates? Also `if blockable` maybe not required?

Thoughts?